packages/backend-core/package.json

```diff
@@ -60,7 +60,7 @@
     ]
   },
   "devDependencies": {
-    "@shopify/jest-koa-mocks": "3.1.5",
+    "@shopify/jest-koa-mocks": "5.0.1",
```

Comment:
How come this one needed bumped? It looks like it wouldn't cause issues

Reply:
unsure, was experiencing build issues with current version, bumped it and it helped solve it - didn't seem to cause issues bumping it up.

Reply:
It was probably because the `tests` dir was being included, which required a usage on this dev dependency as mentioned on the issue description. We could probably get away with not bumping, but it shouldn't hurt. 

There should be a lockfile change associated with this bump however?